structuredheadings/plugin.js

```diff
@@ -31,12 +38,39 @@
         }
       };
 
+      var getCurrentBlock = function (element) {
+        return element.getAscendant({
```

Comment:
cool!   how differently does this behave from elementpath.block? http://docs.ckeditor.com/#!/api/CKEDITOR.dom.elementPath-property-block